src/i18n/messages/nl.json

```diff
@@ -5,7 +5,7 @@
     "originalDefault": "Completed"
   },
   "+DxzBU": {
-    "defaultMessage": "Vul hieronder uw e-mailadres in. U krijgt dan een mail met link om het formulier verder in te vullen. Dit kan op elk apparaat waar u de link opent. De link blijft maximaal {numberOfDays, plural, one {1 dag} other {{numberOfDays} dagen}} actief.",
+    "defaultMessage": "Vul hieronder je e-mailadres in. Je krijgt dan een mail met link om het formulier verder in te vullen. Dit kan op elk apparaat waar je de link opent. De link blijft maximaal {numberOfDays, plural, one {1 dag} other {{numberOfDays} dagen}} actief.",
```

Comment:
Technically `je` as possessive pronoun is incorrect, but saying `Vul hieronder jouw e-mailadres in` sounds/reads even weirder imo, so I think this is fine :stuck_out_tongue: